src/selection.ts

```diff
@@ -1,22 +1,16 @@
-import {Binding} from 'vega';
+import {Binding, Vector2} from 'vega';
 import {SingleDefUnitChannel} from './channel';
-import {FieldName} from './channeldef';
+import {FieldName, Value} from './channeldef';
 import {DateTime} from './datetime';
+import {Dict} from './util';
 import {Color, EventStream} from './vega.schema';
 
 export const SELECTION_ID = '_vgsid_';
 export type SelectionType = 'single' | 'multi' | 'interval';
 export type SelectionResolution = 'global' | 'union' | 'intersect';
 
-export type SelectionInit = boolean | number | string | DateTime;
-export type SelectionInitInterval = [boolean, boolean] | [number, number] | [string, string] | [DateTime, DateTime];
-
-export interface SelectionInitMapping {
-  [key: string]: SelectionInit;
-}
-export interface SelectionInitIntervalMapping {
-  [key: string]: SelectionInitInterval;
-}
+export type SelectionInit = Value | DateTime;
```

Comment:
There is nothing specific about SelectionInit.

I think it's better to call this `ValueOrDateTime` and move it to the same place as `Value`?

Also need to update `links.yml`.

Reply:
I don't think I want to rename these as it makes it much harder to understand what the type is used for. I prefer if the types have some meaning to them. If we change it, it's also not clear how to link since `Dict<ValueOrDateTime>` is not unambiguously the selection init value.